src/main/java/org/apache/commons/beanutils2/locale/converters/StringLocaleConverter.java

```diff
@@ -268,7 +268,7 @@ else if (value instanceof Date) { // java.util.Date, java.sql.Date, java.sql.Tim
      *
      * @throws ConversionException if conversion cannot be performed
      *  successfully
-     * @throws ParseException if an error occurs parsing a String to a Number
+     * @throws IllegalArgumentException if an error occurs parsing a String to a Number
```

Comment:
@garydgregory 
IMO only when `pattern == null` can it be NPE.
but that two functions are in branch `if(pattern != null)` 
so I don't think it can.